README.md

```diff
@@ -35,7 +46,8 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+// Instructor.findone({name: "Andy"})
+//wishlist_items.create({description:"Resin Laying Deer Figurine, Gold""})
 ```
```

Comment:
You need to save the instructor to a variable so you can then access its wishlist items. Currently, you are trying to access a `wishlist_items` variable that doesn't exist.

Also you would want to create the wishlist item asynchronously, making sure it happens only after the instructor has been retrieved from the database.